GooglePlayInstant/Editor/QuickDeploy/WwwRequestInProgress.cs

```diff
@@ -0,0 +1,106 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Runtime.CompilerServices;
+using UnityEditor;
+using UnityEngine;
+
+[assembly: InternalsVisibleTo("GooglePlayInstant.Tests.Editor.QuickDeploy")]
+
+namespace GooglePlayInstant.Editor
+{
+    /// <summary>
+    /// Provides functionality for tracking HTTP requests represented by corresponding WWW instances, as well as
+    /// executing scheduled actions on the responses when the requests are complete.
+    /// </summary>
+    public class WwwRequestInProgress
+    {
+        private readonly WWW _www;
```

Comment:
Please consider:
- Make a separate class encapsulating these 3 fields
- Change the fields to properties with `{ get; private set;}`
- Make that class IDisposable
- Handle www.Dispose() in the new Dispose()